build.yaml

```diff
@@ -0,0 +1,19 @@
+# See https://github.com/dart-lang/build/tree/master/build_web_compilers#configuration
+# Matches previous configuration in pubspec.yaml - transformers - $dart2js
+targets:
+  $default:
+    builders:
+      build_web_compilers|entrypoint:
+        # These are globs for the entrypoints you want to compile.
+        generate_for:
+        - web/**.dart
+        options:
+          compiler: dart2js
```

Comment:
no ddc config?

Reply:
Definitely should standardize on doing dart2js as `--config=release`.

(Or if we add `pub run build_server deploy`, or something)